setup.py

```diff
@@ -14,7 +14,7 @@
 __version__ = version["__version__"]
 
 install_requires = [
-    "openai",
+    "litellm",
```

Comment:
litellm calls openai, and has a light amount of dependencies

Reply:
cc @UweR70

Reply:
Are you sure about removing  "openai" in "setup.py" , line 17? 
Because its need in "completion.py", line 14?!

Reply:
litellm has openai as a dependency

but happy to keep openai as a dependency too to minimize any drastic changes in this PR